middleware.js

```diff
@@ -285,15 +298,15 @@ module.exports = function(options) {
         }
 
         if (sassStats.mtime > cssStats.mtime) { // Source has changed, compile it
-          if (debug) { log('debug', 'modified', cssPath); }
+          log('debug', 'compile', cssPath, 'was modified');
           return compile();
         }
 
         // Already compiled, check imports
         checkImports(sassPath, cssStats.mtime, function(changed) {
           if (debug && changed && changed.length) {
             changed.forEach(function(path) {
-              log('debug', 'modified import %s', path);
+              log('debug', 'compile', path, '(import file) was modified');
```

Comment:
Note that `%s` here was not expanding, now the fourth parameter is interpolated in the flattened log message.